tool/tbot/config/configtemplate_ssh.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/teleport/tool/tbot/identity"
+	botutils "github.com/gravitational/teleport/tool/tbot/utils"
+	"github.com/gravitational/trace"
+	"golang.org/x/crypto/ssh"
+)
+
+const DEFAULT_PROXY_PORT = 3023
```

Comment:
Do we not already have a constant for this?

Reply:
Ah, yup, `defaults.SSHProxyListenPort`